common/djangoapps/util/views.py

```diff
@@ -44,6 +47,21 @@ def inner(request, *args, **kwargs):
     return inner
 
 
+def require_global_staff(func):
```

Comment:
shouldn't we just move the original into "common" so both lms and csm can use it?